### ISSUE TITLE
Moving a tab that is loading results in the tab forever showing loading indicators

### ISSUE THREAD

**Issue by echosa** [brave/browser-laptop#6957]
- Did you search for similar issues before submitting this one?

Yes

- Describe the issue you encountered:

If you go to a URL in a tab, then move that tab while the page is still loading, the `X` next to the URL bar that normally stops the loading and the spinning circle in the tab favicon area both remain indefinitely, even after the page is loaded. Clicking the `X` does nothing.

- Platform (Win7, 8, 10? macOS? Linux distro?):

OS X 10.11.6

- Brave Version (revision SHA):

0.13.0 (8ca0127)

- Steps to reproduce:
    1. Have multiple tabs open.
    2. Go to a URL in one of the tabs.
    3. While the page is loading, move the tab to a different position.
    
- Actual result:

The `X` and spinner should clear when the page is loaded.

- Expected result:

The `X` and spinner don't clear and remain indefinitely.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?

Yes

- Is this an issue in the currently released version?

Yes

- Can this issue be consistently reproduced?

Yes

- Screenshot if needed:

<img width="78" alt="screen shot 2017-01-31 at 3 54 00 pm" src="https://cloud.githubusercontent.com/assets/133786/22486214/3ab1f76e-e7ce-11e6-830a-72852279ed0c.png">

**Comment by cezaraugusto** [comment]
unfortunately still reproducible when dragging tab into a new window. Not addressed with #10691 so removing from sprint/1. /cc @zentagonist

**Comment by echosa** [comment]
I can no longer reproduce this issue. Seems like it has been fixed somewhere along the way. Closing the issue. 

Current Brave version:

```
Brave: 0.21.18 
V8: 6.4.388.41 
rev: 580be782bc37ed8580dd22f79e1d2d355fddfb89 
Muon: 4.8.2 
OS Release: 17.4.0 
Update Channel: Release 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 64.0.3282.140
```
